DataScience/GenevaLogger.py

```diff
@@ -0,0 +1,48 @@
+from fluent import asyncsender as sender
+import json
+import sys
+import traceback
+
+class Logger:
+    app_id = ""
+    job_id = ""
+
+    @staticmethod
+    def create_logger(app_id, job_id):
+        sender.setup('microsoft.cloudai.personalization', host='localhost', port=24224)
+        Logger.app_id = app_id
+        Logger.job_id = job_id
+
+    @staticmethod
+    def __log(level, msg, output=sys.stdout, tag='log', **kwargs):
+        try:
+            print(msg, file=output)
+            base_log = {'level': level, 'message': msg, 'appId': Logger.app_id, 'jobId': Logger.job_id}
+            log_content = {**base_log, **kwargs}
+            logger = sender.get_global_sender()
+            if not logger.emit(tag, log_content):
+                print(logger.last_error)
+                logger.clear_last_error()
+        except Exception as e:
+            print("Error while logging: {}".format(e))
+
+    @staticmethod
+    def info(msg: str, **kwargs):
+        Logger.__log('INFO', msg, **kwargs)
```

Comment:
hmm, two things here. I'm using this [article](https://radek.io/2011/07/21/static-variables-and-methods-in-python/#:~:text=%20Static%20variables%20and%20methods%20in%20Python%20,defining%20static%20methods...%203%20Sources.%20%20More%20) as reference:

1.  All class variables in Python are also static, so I think it's appropriate to access them from static methods, no? 
2. Making these class methods forces me to instantiate a Logger object and pass it around to all methods/modules that need logging, which is what I'm trying to avoid

Reply:
# 1 there isn't an issue with access, but there are other issues with referencing a class variable in a static method.
# 2 isn't true. classmethods are on the class and not the instance. There is no need to instantiate an object to reference them.

This short example I think illustrates the issue.
```
class Foo(object):
    my_name = "Foo"

    @classmethod
    def my_class_method(cls):
        print(f'Hello {cls.my_name}!')

    @staticmethod
    def my_static_method():
        print(f'Hello {Foo.my_name}!')


class Bar(Foo):
    my_name = "Bar"

if __name__ == "__main__":
    Foo.my_class_method()
    Foo.my_static_method()
    Bar.my_class_method()
    Bar.my_static_method()

# Output
# Hello Foo!
# Hello Foo!
# Hello Bar!
# Hello Foo!
```
Notice that for the class method Bar can override the behavior, but not for the static method.

Reply:
I realize above that I was mistaking a class method for an instance method. Thanks for the explanation and I'll take your advice